### DISCUSSION TITLE
ESLINT will drop formatting in 9.x

### DISCUSSION THREAD

**Discussion by jankapunkt** [node-oauth/node-oauth2-server#262] [General]
ESLINT 9 will drop formatting (`--fix`) so we should think about a formatter, if we want one or if we keep the formatting to individuals. Can we even simply use prettier to comply with our current eslint config?

**Comment by shrihari-prakash** [comment]
@jankapunkt , we could also look at using EditorConfig as it is cross IDE compatible and can have the same set of extensive rules as ESLINT.

**Reply by jankapunkt** [reply]
@shrihari-prakash let's keep this in mind. Im currently monitoring the discussions around eslint and related packages to see what the trends are there. Again - we could also decide to simply not include formatting and leave this to each of us, since the CI will anyway throw if things don't comply with the ESLINT rules
